diff --git a/src/spawn-enums.c b/src/spawn-enums.c
index 17bf5a0..488ad05 100644
--- a/src/spawn-enums.c
+++ b/src/spawn-enums.c
@@ -29,6 +29,12 @@
 #include <stdio.h>
 #include <string.h>
 
+// Define missing variables for Debian 10
+#define SCMP_ACT_KILL_PROCESS	0x80000000U
+#define SCMP_ACT_KILL_THREAD	0x00000000U
+#define SCMP_ACT_LOG		0x7ffc0000U
+#define SCMP_FLTATR_CTL_LOG     6
+
 // for seccomp syscall use seccomp_syscall_resolve_name("syscallname")
 const nsKeyEnumT nsScmpFilter [] = {
     {"SCMP_FLTATR_ACT_DEFAULT", SCMP_FLTATR_ACT_DEFAULT},
